docs/developer-guide/ksqldb-reference/select-push-query.md

```diff
@@ -29,17 +29,23 @@ Description
 
 Push a continuous stream of updates to the ksqlDB stream or table. The result of
 this statement isn't persisted in a Kafka topic and is printed out only in
-the console. To stop the continuous query in the CLI press Ctrl+C.
-Note that the WINDOW clause can only be used if the `from_item` is a
-stream.
+the console, or returned to the client. To stop a push query started in the CLI press Ctrl+C.
 
-Push queries enable you to query a materialized view with a subscription to
-the results. Push queries emit refinements to materialized views, which enable
+Execute a push query via the CLI or by sending an HTTP request to the ksqlDB REST API, and
+the API sends back a chunked response of indefinite length.
+
+Push queries enable you to subscribe to changes, which enable

```

Comment:
I've changed this from:

> Push queries enable you to query a materialized view

As that would suggest you can only perform push queries on MVs.... which will suggest tables to most users, while ksqlDB supports push queries on streams too.